Modules/ITS/src/ITSTrackTask.cxx

```diff
@@ -113,7 +113,8 @@ void ITSTrackTask::monitorData(o2::framework::ProcessingContext& ctx)
       vPhi.emplace_back(out.getPhi());
     }
 
-    tClusterMap->Fill();
+    if (mDoTTree)
```

Comment:
I let the PR merge automatically once the tests pass to allow it to go to the nightly, but please add braces in these if statements on the next opportunity.